google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -382,6 +386,10 @@ static boolean isEnableTransactionalConnectionStateForPostgreSQL() {
                       ROUTE_TO_LEADER_PROPERTY_NAME,
                       "Should read/write transactions and partitioned DML be routed to leader region (true/false)",
                       DEFAULT_ROUTE_TO_LEADER),
+                  ConnectionProperty.createBooleanProperty(
+                      ENABLE_END_TO_END_TRACING_PROPERTY_NAME,
+                      "Should we enable end to end tracing (true/false)",
```

Comment:
Create a more descriptive description that includes what this property actually does. Something like `Enable end-to-end tracing (true/false) to generate traces for both the time that is spent in the client, as well as time that is spent in the Spanner server. This option can only be used in combination with an exporter that exports the traces to Google Cloud Trace.` (I'm not sure about the latter, but I thought that this was a limitation, right?)

Reply:
Note that we are planning on automatically generating documentation from these descriptions, so they should really provide information about what it does, and what limitations might apply.

Reply:
Updated the description. Please review